src/graph/LookupExecutor.cpp

```diff
@@ -0,0 +1,499 @@
+    }
+}
+
+void LookupExecutor::processEdgeResult(EdgeRpcResponse &&result) {
+    auto all = result.responses();
+    std::shared_ptr<SchemaWriter> outputSchema;
+    std::unique_ptr<RowSetWriter> rsWriter;
+    std::shared_ptr<ResultSchemaProvider> schema = nullptr;
+    for (auto &resp : all) {
+        if (!resp.__isset.rows || resp.get_rows() == nullptr || resp.rows.empty()) {
+            continue;
+        }
+        if (outputSchema == nullptr) {
+            if (!yields_.empty()) {
+                schema = std::make_shared<ResultSchemaProvider>(*(resp.get_schema()));
+            }
+            outputSchema = std::make_shared<SchemaWriter>();
+            outputSchema->appendCol("edgeSrc", nebula::cpp2::SupportedType::VID);
+            outputSchema->appendCol("edgeDst", nebula::cpp2::SupportedType::VID);
+            outputSchema->appendCol("edgeRank", nebula::cpp2::SupportedType::INT);
+            setOutputYields(outputSchema.get(), schema);
+            rsWriter = std::make_unique<RowSetWriter>(outputSchema);
+        }
+        for (const auto& data : *resp.get_rows()) {
+            RowWriter writer(outputSchema);
+            const auto& edge = data.get_key();
+            writer << edge.get_src() << edge.get_dst() << edge.get_ranking();
+            for (auto& column : returnCols_) {
+                auto reader = RowReader::getRowReader(data.get_props(), schema);
+                auto res = RowReader::getPropByName(reader.get(), column);
+                if (!ok(res)) {
+                    LOG(ERROR) << "Skip the bad value for prop " << column;
+                    continue;
+                }
+                auto&& v = value(std::move(res));
+                auto status = Collector::collect(v, &writer);
+                if (!status.ok()) {
+                    LOG(ERROR) << "Collect prop error: " << status;
+                }
+            }
+            rsWriter->addRow(writer.encode());
+        }
+    }  // for `resp'
+    finishExecution(std::move(rsWriter));
+}
+
+void LookupExecutor::processVertexResult(VertexRpcResponse &&result) {
+    auto all = result.responses();
+    std::shared_ptr<SchemaWriter> outputSchema;
+    std::unique_ptr<RowSetWriter> rsWriter;
+    std::shared_ptr<ResultSchemaProvider> schema = nullptr;
+    for (auto &resp : all) {
+        if (!resp.__isset.rows || resp.get_rows() == nullptr || resp.rows.empty()) {
+            continue;
+        }
+        if (outputSchema == nullptr) {
+            if (!yields_.empty()) {
+                schema = std::make_shared<ResultSchemaProvider>(*(resp.get_schema()));
+            }
+            outputSchema = std::make_shared<SchemaWriter>();
+            outputSchema->appendCol("VertexID", nebula::cpp2::SupportedType::VID);
+            setOutputYields(outputSchema.get(), schema);
+            rsWriter = std::make_unique<RowSetWriter>(outputSchema);
+        }
+        for (const auto& data : *resp.get_rows()) {
```

Comment:
Perhaps you have missed the vid column when output. See example below, if type this more times, the console would crash.
![image](https://user-images.githubusercontent.com/13706157/74010424-6da06c80-49c0-11ea-97cf-e1d7615c204b.png)

Reply:
> Perhaps you have missed the vid column when output. See example below, if type this more times, the console would crash.
> ![image](https://user-images.githubusercontent.com/13706157/74010424-6da06c80-49c0-11ea-97cf-e1d7615c204b.png)

fixed